storm/conf.yaml.example

```diff
@@ -0,0 +1,33 @@
+init_config:
+  # Default environment to use for all instances
+  # environment: production
+  #
+  # Default Storm API endpoint (typically this is the UI server on the UI port)
+  #   ex. if storm ui is running on the localhost on port 9005
+  # server: localhost:9005
+  #
+  # Use HTTPS by default
```

Comment:
we could remove that setting and let the user set it through the server (ie: `server: http://localhost:9005`)